src/Routes/__tests__/GSheetImport.test.ts

```diff
@@ -39,7 +39,7 @@ describe("GSheetImport", () => {
 
     it("Calls updateDatabase with downloaded rows data", async () => {
       req.body = {
-        spreadSheetID: "123",
+        spreadSheetID: "test_id123",
```

Comment:
Is this what the ids actually look like? If not, using something more representative might serve as good documentation. (Personally, I have no idea where to look for a document's id.)

Reply:
No, the spreadsheet id are a usually longer and can be found in the URL. `https://docs.google.com/spreadsheets/d/<spreadsheetId>/edit#gid=<sheetId>`

Good point though, I can add some documentation around this